src/app/utils/sample-url-sanitizer.ts

```diff
@@ -0,0 +1,37 @@
+function redactPii(str: string)
+{
+  const piiRegex = [
```

Comment:
@MIchaelMainer  can you give this a look over to make sure there's no other formats that may be needed.

Reply:
We shouldn't do this. Explanation in my review comment in telemetry.ts.